### PR TITLE
Feature: Pickup Action Art and Sound Polish [MTT-4615] [MTT-4304]

### PR DESCRIPTION
**Pull request by jilfranco-unity**
### Description
A pass on the pick up action to add in some VFX and audio for when the player picks up and puts down the pot!

Here's a video of the effects!

https://user-images.githubusercontent.com/89089503/189434040-7308769c-6037-4679-83f4-fc145cd20c52.mp4

### Issue Number(s)
Jira tasks [MTT-4615](https://jira.unity3d.com/browse/MTT-4615) and [MTT-4304](https://jira.unity3d.com/browse/MTT-4304)

### Contribution checklist
 - [ ] Tests have been added for boss room and/or utilities pack
 - [ ] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

File: Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
+using System;
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Server
+{
+    /// <summary>
+    /// Component to simply play a descending animation when this NetworkObject's parent NetworkObject changes.
+    /// </summary>
+    public class ClientPickUpPotVisualizer : NetworkBehaviour
```
Comment (SamuelBellomo) [inline Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs:10]: It's not just "visualizer" if there's sound as well :/

File: Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
+using System;
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Server
```
Comment (SamuelBellomo) [inline Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs:5]: server namespace for client scripts?

**Review by SamuelBellomo — APPROVED** [review]
sounds are cute :) are they accelerated imp melee?

File: Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs (hunk 1 above)
Comment (jilfranco-unity) [reply Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs:10]: would something like ClientPickUpPotEffects work better?

Comment (jilfranco-unity) [reply Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs:10]: tho I'll also raise that the ClientTossedItemVisualizer script also has an AudioSource reference too 😅

**Review by LPLafontaineB — COMMENTED** [review]
Very nice! However I think the sound is a bit loud in comparison with the attack sounds? Or was that the intent?

File: Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs (hunk 1 above)
Comment (SamuelBellomo) [reply Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs:10]: 🚀

Comment (SamuelBellomo) [reply Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs:10]: > tho I'll also raise that the ClientTossedItemVisualizer script also has an AudioSource reference too 😅

Might be worth updating it as well @fernando-cortez

**Comment by jilfranco-unity** [comment]
> Very nice! However I think the sound is a bit loud in comparison with the attack sounds? Or was that the intent?

I don't want it to be too loud, I'll take another listen and adjust!

**Review by LPLafontaineB — APPROVED** [review]

File: Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs (hunk 1 above)
Comment (fernando-cortez) [reply Assets/Scripts/Gameplay/GameplayObjects/ClientPickUpPotVisualizer.cs]: Added a task for this! [MTT-4654](https://jira.unity3d.com/browse/MTT-4654).

**Review by fernando-cortez — APPROVED** [review]
Just needs a formatting pass, but gtg!

**Review by LPLafontaineB — APPROVED** [review]

**Review by fernando-cortez — APPROVED** [review]

**Pull request merged**